brands/amenity/social_centre.json

```diff
@@ -1,20 +1,77 @@
 {
-  "amenity/social_centre|American Legion": {
+  "amenity/social_centre|American Legion Hall": {
     "countryCodes": ["us"],
+    "matchNames": ["american legion"],
     "tags": {
+      "alt_name": "American Legion Hall",
       "amenity": "social_centre",
       "brand": "American Legion",
       "brand:wikidata": "Q468865",
       "brand:wikipedia": "en:American Legion",
-      "name": "American Legion",
+      "name": "American Legion Hall",
```

Comment:
Did you mean to add "Hall" here and also in the key above?
It's ok to be in `alt_name`.

Reply:
I changed it for consistency with the other entries – `name=Benevolent and Protective Order of Elks` is a real mouthful compared to `name=Elks Lodge`, which can get the mapper started for a locally appropriate lodge name and number. But maybe the entries shouldn’t be consistent with each other, because it is common to see an American Legion Hall signposted as just “American Legion”, whereas it’s quite uncommon to see “Benevolent and Protective Order of Elks” anywhere out in the field.

Reply:
Ok seems fine.. I don't really have a strong preference, and encouraging consistency makes sense.